data/35e/wizards_of_the_coast/supplement/miniatures_handbook/mh_classes.lst

```diff
@@ -4,7 +4,7 @@ CLASS:Healer
 
 CLASS:Marshal	MAXLEVEL:20
 CLASS:Marshal	HD:8	TYPE:Base.PC
-CLASS:Marshal	FACT:Abb|	FACT:ClassType|
+CLASS:Marshal	FACT:Abb|Mshl	FACT:ClassType|Prestige
```

Comment:
20th level progression classes are not Prestige, this would be a Base ClassType.